myproject/blog/urls.py

```diff
@@ -8,5 +8,7 @@
      path("blogs/",views.BlogList.as_view(), name="blog_list"),
      path("blogger/<int:pk>/",views.AuthorProfile.as_view(), name="author"),
      path("blogs/<int:pk>/",views.BlogDetail.as_view(), name="blogdetail"),
-     path("bloggers/",views.BloggerList.as_view(), name="bloglist"), 
+     path("bloggers/",views.BloggerList.as_view(), name="bloggerlist"), 
+     path("createauthor/",views.CreateAuthor.as_view(), name="create_author"), 
```

Comment:
Please be consistent with the naming conventions, if you have used bloggers, then this should be `create-blogger` instead of createauthor.